db/migrations/V5__create_kanji_schema.sql

```diff
@@ -44,4 +44,4 @@ create view kanji.on_reading as select id as kanji_character_id, character, unne
 create view vocabulary.vocabulary_characters as select distinct on (vocabulary_id, character) vocabulary_id, unnest(regexp_split_to_array(value, '')) as character from vocabulary.spelling;
 
 -- Unlike vocabulary.characters which is vocab focused, this one includes the spelling_id so that filters/joins to spellings can occur.
-create view vocabulary.spelling_characters as select distinct on (vocabulary_id, spelling_id, character) vocabulary_id, id as spelling_id, unnest(regexp_split_to_array(value, '')) as character from vocabulary.spelling;
\ No newline at end of file
+create view vocabulary.spelling_characters as select distinct on (vocabulary_id, spelling_id, character) vocabulary_id, id as spelling_id, unnest(regexp_split_to_array(value, '')) as character from vocabulary.spelling;
```

Comment:
Added a blank line because the RubyMine search doesn't show the last line so well (and occasionally there is text processing tooling that likes an empty line at the end of files, which is why its a recommended practice in some programming languages.)